source/docs/articles/sites/domains/setting-up-a-domain-with-godaddy.md

```diff
@@ -1,44 +1,27 @@
- ![Activate your domain](/source/docs/assets/images/desk_images/49496.png)
-## Review Your Cart
-
This is the last time you will be able to make any corrections before you become the owner a brand new domain. You should review the cart to make sure everything is correct before you proceed.  
- ![Review your order](/source/docs/assets/images/desk_images/49497.png)
-## Go to Your Account Page
-
After you complete the purchase process you can begin configuring your domain and you will be presented with the order summary. Click the **My Account** button that will take you to your global GoDaddy Account page.
- ![Sign in to your Godaddy account](/source/docs/assets/images/desk_images/49498.png)
-#### Launch the Domain Control Panel
-
Now you can get to the next stage and click on the **Launch** button which will give you access to your GoDaddy domain control panel. We have instructions if you are using either the [Old](#old-dns-dashboard) or the [New](#new-dns-dashboard) versions of the DNS manager.  
- ![Launch your control panel](/source/docs/assets/images/desk_images/49499.png)  
-### Old DNS Dashboard
-Click the name of the domain that you want to edit and this will take you to the domain manager.  
- ![](/source/docs/assets/images/desk_images/49521.png)  
-Next, you will see the Zone summary screen that will give you the information associated with this domain. Click on the **Launch** button to be taken to the GoDaddy Zone config tool.  
- ![Edit the DNS information](/source/docs/assets/images/desk_images/49502.png)
-### New DNS Dashboard
-
If you are using the New Standard DNS dashboard, you can click on the **Edit Zone** link which will take you directly to the GoDaddy Zone configuration tool.  
- ![DNS dashboard](/source/docs/assets/images/desk_images/49520.png)
-#### Setting Up the Zone Record
-The last step on GoDaddy is to make sure you have correctly setup your A record or CNAME for your domain. Make sure you have put in the I.P. address or the CNAME to point your domain to Pantheon. You can get more information about this on our [Going Live](/docs/articles/going-live/) documentation page. 

+
+Visit GoDaddy.com to search for the domain name you want and add it to your cart. If you have more than one domain, you will be required to configure the A records or CNAME for each one.
+
+If you need to add email to your new domain, you can add it at this time. Pantheon does not provide email hosting services.
+
+After you've purchased your domain name, follow the instructions below.  
+
+## Configure Domain Settings
+
1. Sign in to your account at GoDaddy.com.
+2. Click **Manage My Domains**, then **Manage DNS**.
+3. Enter the **A record** or **CNAME** for your domain. Make sure to enter the IP address or the CNAME to point your domain to Pantheon. For more information, see [Going Live](/docs/articles/going-live/). 
```

Comment:
@rachelwhitton So you're saying it was missing the gear icon step? I thought that was obvious but we should add it just to be clear. thanks!